cypress/e2e/pages/topicPage/tests.js

```diff
@@ -133,10 +137,12 @@ export default ({ service, pageType, variant }) => {
         }
       });
     });
+
     describe(`Pagination`, () => {
       it('should show pagination if there is more than one page', () => {
-        // First return to the topics page. Last test has page on article
-        cy.go('back');
+        if (pageCount === undefined) {
```

Comment:
Do we need this conditional here to skip the test?

If we do, do we want to have a log to say why the pageCount might be undefined? It could be undefined because we are on the wrong page, like how we saw when we looked at the test failures from being still on an article page. You visiting the pages in the BeforeEach in the describe block above should ensure this doesn't happen. Is there another reason why it could happen, and if so, should be log something so we don't start skipping the first test without knowing? (although, the following tests would fail anyway if the pagination isn't present).

Reply:
I don't think we need this - we check below whether pageCount is > 1, and will only run the pagination assertions if that's the case. Let's remove this.